src/Fixer/LanguageConstruct/SingleSpaceAfterConstructFixer.php

```diff
@@ -339,4 +343,16 @@ private function isMultilineExtendsOrImplementsWithMoreThanOneAncestor(Tokens $t
 
         return false;
     }
+
+    private function isMultilineConstant(Tokens $tokens, int $index): bool
+    {
+        $scopeEnd = $tokens->getNextTokenOfKind($index, [';']);
```

Comment:
`$scopeEnd = $tokens->getNextTokenOfKind($index, [';', [T_CLOSE_TAG]]);`
this will never be `null`

Reply:
Thanks!
Fixed by following commit :)
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/6167/commits/c23255a34491ca4d9c8c77906ef3e62c7cc02aeb